src/Endpoints/MethodEndpoint.php

```diff
@@ -44,7 +45,25 @@ public function all(array $parameters = [])
      */
     public function allActive(array $parameters = [])
     {
-        return parent::rest_list(null, null, $parameters);
+        $url = "methods/all" . $this->buildQueryString($parameters);
+        $result = $this->client->performHttpCall("GET", $url);
```

Comment:
why not use the existing `allAvailable()` method here which already fetches the `methods/all` endpoint

```suggestion
        $result = $this->allAvailable($parameters);
```